packages/core/types/src/modules/documents/service-instance.ts

```diff
@@ -71,4 +80,22 @@ export type ServiceInstance<
     ) => Result.DiscardDraft<TContentTypeUID, TParams>,
     undefined
   >;
+
+  updateComponents: <
+    TUID extends Common.UID.Schema,
+    TData extends EntityService.Params.Data.Input<TUID>
```

Comment:
Do we actually need this generic parameter?
By using it like this we're not constraining data to be exactly Input<TUID>, but any object that extends it.

Is this the wanted behavior? (<=> do we need data type to be inferred for a specific reason?)